src/APIM_ARMTemplate/apimtemplate/Extractor/EntityExtractors/ProductAPIExtractor.cs

```diff
@@ -0,0 +1,436 @@
+﻿using System;
```

Comment:
is there any reason that you create this file and re-add these operations?

Reply:
Removed duplication